app/views/shared/check_answers/_housing_benefit.html.erb

```diff
@@ -1,4 +1,3 @@
-<h2 class="govuk-heading-l">Housing Benefit</h2>
```

Comment:
Accessibility: does removing this `h2` break the heading order or is there a relevant heading in the parent partial?

Reply:
Having this h3 here matches the other heading levels for the Income section. Overall though, the heading order on this page is quite broken due to partials being reused in multiple different places but this will be addressed in the future work on this page. The removal of this heading in particular was a specific request from content/design hence why it's included here